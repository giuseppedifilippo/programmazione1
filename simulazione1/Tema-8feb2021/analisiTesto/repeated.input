ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
ciao
