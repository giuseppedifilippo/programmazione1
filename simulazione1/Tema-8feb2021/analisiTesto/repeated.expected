ciao : 40
40 40 40
[]
